rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -975,4 +977,10 @@ stream {
 
         {{ end }}
 
+        {{ if $all.DynamicConfigurationEnabled}}
+        log_by_lua_block {
+            balancer.after_balance()
+        }
+        {{ end }}
```

Comment:
I think this should be in https://github.com/Shopify/ingress/pull/19/files#diff-980db9e4b88704f12338bd074839f94eR941